src/routers/post.js

```diff
@@ -0,0 +1,53 @@
+const { FastifyPluginCallback } = require("fastify");
+
+const store = require("../store");
+const errHandler = (err) => {console.log(err);}
+
+/**
+ * @type {FastifyPluginCallback}
+ */
+
+const postPlugin = (fastify, opts, done) => {
+    fastify.post("/pillars", async (req, reply) => {
+        try {
+            content = req.query;
+            await store.postPillar(content);
+            reply.send("Added " + content.name + " to the pillars database");
+        }
+        catch (err) {
+            console.log(err);
+            reply.send(err);
+        }
+    });
+    fastify.post("/courses", async (req, reply) => {
+        try {
+            content = req.query;
+            await store.postCourse(content);
+            reply.send("Added " + content.name + " to the courses database");
```

Comment:
responses to post requests should be a JSON of the resource created, which acts like a confirmation of creation. Or if you want to be asolutely correct, the response should have a 201 HTTP code, even though I haven't seen that out in the wild often.

https://stackoverflow.com/questions/1226810/is-http-post-request-allowed-to-send-back-a-response-body